<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDayDialog</class>
 <widget class="QDialog" name="SetDayDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>192</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>192</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>192</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modifier un jour</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px</string>
   </property>
   <property name="text">
    <string>Enregistrer la modification</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="start_edit">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>69</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="time">
    <time>
     <hour>9</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QTimeEdit" name="end_edit">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>69</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="time">
    <time>
     <hour>17</hour>
     <minute>15</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="break_edit">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>69</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="value">
    <double>0.450000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>29</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrivé</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>29</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>29</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Départ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>123</y>
     <width>143</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>59</width>
        <height>59</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>59</width>
        <height>59</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/clock/pictures/clock-8-128.png) 0 0 0 0 stretch stretch;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font-size: 14px;</string>
        </property>
        <property name="text">
         <string>Temps total</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="total_label">
        <property name="styleSheet">
         <string notr="true">font-size: 28px;
color: white;</string>
        </property>
        <property name="text">
         <string>0h</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="validateButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px</string>
   </property>
   <property name="text">
    <string>Définir l'horaire comme valide</string>
   </property>
  </widget>
  <widget class="QFrame" name="titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(204, 22, 0, 255), stop:1 rgba(217, 0, 112, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="window_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-1</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-size: 11px;
background: none;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>377</x>
     <y>0</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe MDL2 Assets</family>
    </font>
   </property>
   <property name="text">
    <string></string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="window_border_left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(204, 22, 0, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="window_border_bottom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>419</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 rgba(204, 22, 0, 255), stop:1 rgba(217, 0, 112, 255));</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="window_border_right">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>0</y>
     <width>2</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgba(217, 0, 112, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SetDayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>SetDayDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
